webpack.config.js

```diff
@@ -67,5 +68,10 @@ module.exports = (env = {}) => {
         jquery: path.resolve(__dirname, 'node_modules/jquery'),
       },
     },
+    plugins: [
+      new webpack.DefinePlugin({
+        __WEBPACK_REGION__: `'${env.region.toUpperCase()}'`,
```

Comment:
Using template literal in that case doesn't have any benefits right?

Is enough?

```suggestion
        __WEBPACK_REGION__: env.region.toUpperCase(),
```

Reply:
In fact, Webpack is going to replace `__WEBPACK_REGION__` by the value, so, in your case, it will replace:

```javascript
myObject.method(__WEBPACK_REGION__);
```

by

```javascript
myObject.method(EU);
```

EU will be considered as variable, so EU will be undefined ^^

If we let the magic quote, we will get:

```javascript
myObject.method('EU');
```

I suggest you to checkout the branch and test if you have any doubt. Will be happy to discuss about that if needed :)

Reply:
Got it! Thanks!